RxCocoa/Traits/SharedSequence/SharedSequence+Concurrency.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  SharedSequence+Concurrency.swift
+//  RxCocoa
+//
+//  Created by Shai Mishali on 22/09/2021.
+//  Copyright © 2021 Krunoslav Zaher. All rights reserved.
+//
+
+#if swift(>=5.5) && canImport(_Concurrency) && !os(Linux)
+import Foundation
+
+// MARK: - Shared Sequence
+@available(macOS 12, iOS 15, tvOS 15, watchOS 8, *)
+public extension SharedSequence {
+    /// Allows iterating over the values of this Shared Sequence
+    /// asynchronously via Swift's concurrency features (`async/await`)
+    ///
+    /// A sample usage would look like so:
+    ///
+    /// ```swift
+    /// for await value in driver.values {
+    ///     // Handle emitted values
+    /// }
+    /// ```
+    @MainActor var values: AsyncStream<Element> {
```

Comment:
Why values should be accessed on main thread ?

Reply:
`Driver` and `Signal` emit on the main thread:
https://github.com/ReactiveX/RxSwift/blob/main/RxCocoa/Traits/Driver/Driver.swift#L15